iib/workers/config.py

```diff
@@ -81,6 +86,7 @@ class TestingConfig(DevelopmentConfig):
     iib_greenwave_url = 'some_url'
     iib_omps_url = 'some_url'
     iib_request_logs_dir = None
+    iib_dogpile_expiration_time = 5
```

Comment:
Don't we want caching disabled in testing by default?

Reply:
There is a test for caching, so I guess no?

Reply:
In this case, I would enable caching just for the test that tests caching. I imagine having caching enabled for all tests could produce unexpected results based on things being mocked.